test/smoke_test/run_ac_smoke_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+. ./utils.sh
+
+omz_downloader --output_dir working_dir_smoke --cache_dir cache_dir_smoke      --name=Sphereface
+omz_converter  --output_dir working_dir_smoke --download_dir working_dir_smoke --name=Sphereface --precisions FP32
+
+# prepare reduced LFW dataset (only people with name starting with A)
```

Comment:
Do you have link with this instruction? if yes, please also add in comment

Reply:
No, I haven't